tests/data/html5links.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <title>SiteMaster testing site</title>
+</head>
+<body>
+<div>
+    This site just contains a few pages to help with integration testing of SiteMaster
+    <a href="https://bulletin.unl.edu/undergraduate/">Regular link</a>
+    <a href="https&colon;&sol;&sol;bulletin&period;unl&period;edu&sol;undergraduate&sol;">html5 encoded link</a>
+    <a href="https://bulletin.unl.edu/undergraduate/?u=http%3A%2F%2Fexample.com%2Ftest%0D%0A">Regular link with a url as a parameter</a>
+    <a href="https&colon;&sol;&sol;bulletin&period;unl&period;edu&sol;undergraduate&sol;&quest;u&equals;http&percnt;3A&percnt;2F&percnt;2Fexample&period;com&percnt;2Ftest&percnt;0D&percnt;0A">html5 encoded with a param</a>
```

Comment:
Hahaha. Isn't this ridiculously hard to read !? I really should adjust the escaping.